.env.sample

```diff
@@ -14,3 +14,11 @@ IDEMPOTENT_DOCKER=1
 
 # Test credentials to be used in tests
 NA_TESTS_CREDENTIALS={"default": {"email": "name@example.com", "token": "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"}}
+
+# Access token used to download pre-build libtelio, libdrop and libmoose binaries
+PERSONAL_ACCESS_TOKEN="<your-token-here>"
+
+# libtelio, libdrop, libmoose release pages
+LIBTELIO_RELEASES_URL="<libtelio-releases-url-here>"
+LIBDROP_RELEASES_URL="<libdrop-releases-url-here>"
+LIBMOOSE_RELEASES_URL="<libmoose-releases-url-here>"
```

Comment:
Not relevant for OSS builds. Can be migrated to internal knowledge base

Reply:
True. I removed it. I'll update internal knowledge base after this gets merged to main